src/sidebar/store/modules/toast-messages.js

```diff
@@ -98,6 +98,20 @@ function hasMessage(state, type, text) {
   });
 }
 
+/**
+ * @typedef ToastMessagesStore
+ *
+ * // Actions
+ * @prop {typeof addMessage} addToastMessage
+ * @prop {typeof removeMessage} removeToastMessage
+ * @prop {typeof updateMessage} updateToastMessage
+ *
+ * // Selectors
+ * @prop {(id: string) => (Object|undefined)} getToastMessage
+ * @prop {() => Object[]} getToastMessages
```

Comment:
It would be good to introduce a typedef documenting what properties a toast message has. I don't think we have a suitable typedef anywhere yet. The closest thing to a definition is here: https://github.com/hypothesis/client/blob/f93ea8275b996794db98880efa1e25363b254de7/src/sidebar/services/toast-messenger.js#L62.